plugins/civmodcore-paper/src/main/java/vg/civcraft/mc/civmodcore/commands/ChunkMetaCommand.java

```diff
@@ -0,0 +1,33 @@
+package vg.civcraft.mc.civmodcore.commands;
+
+import co.aikar.commands.BaseCommand;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Subcommand;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.command.CommandSender;
+
+@CommandAlias("cmc")
+public class ChunkMetaCommand extends BaseCommand {
+
+	private static boolean ENABLE_CHUNK_META_LOGS = false;
+
+	public static boolean chunkMetaLogsEnabled() {
+		return ENABLE_CHUNK_META_LOGS;
+	}
+
+	@Subcommand("chunkmeta")
```

Comment:
I don't think this is intuitive as a toggle, the command should be something like `/cmc togglechunkmetalogs` or `/cmc chunkmetalogs true`

Reply:
I'll change that